source/js/components/share-button-group/share-button-group.scss

```diff
@@ -23,24 +22,35 @@
       display: flex;
       flex: 1;
 
+      &:not(:last-child) {
+        margin-right: $margin;
+      }
+
       &:last-child {
         .btn:last-child {
           margin-right: 0;
         }
       }
     }
 
-    @media (max-width: $bp-md) {
+    @include media-breakpoint-down("sm") {
```

Comment:
I find using this Boostrap mixin less confusing for `max-width` media query:

`@media (max-width: $bp-md) {` to target screen sizes of `sm`  and smaller
vs
`@include media-breakpoint-down("sm") {`

Reply:
Hm, does it behave the same though? max-width: medium is not the same as "small and below", there's a gap in between those two values.

Reply:
it's the same here. `$bp-md` is the minium width of Bootstrap defined "medium device". So `(max-width: $bp-md)` essentially means "small and below"